src/app/pages/charging-stations/charging-limit/charging-profile-limit/charging-station-charging-profile-limit-schedule-table-data-source.ts

```diff
@@ -23,12 +24,17 @@ export class ChargingStationChargingProfileLimitScheduleTableDataSource extends
     public translateService: TranslateService,
     private datePipe: AppDatePipe,
     private decimalPipe: AppDecimalPipe,
-    private unitPipe: AppUnitPipe
+    private unitPipe: AppUnitPipe,
+    private centralServerNotificationService: CentralServerNotificationService,
   ) {
     super(spinnerService, translateService);
     this.initDataSource();
   }
 
+  public getDataChangeSubject(): Observable<ChangeNotification> {
+    return this.centralServerNotificationService.getSubjectChargingProfiles();
```

Comment:
The data does not come from the DB (loadData) but provided by the external controller.
This DS is not linked to the DB.
This subject has to be changed in charging-station-charging-profile-limit.component.ts controller.

Reply:
It's just a starting point to have refresh from the DB on the charging profile schedule. It's currently not used anyway. A meeting with Tobias will be needed to do it properly.

Reply:
> It's just a starting point to have refresh from the DB on the charging profile schedule. It's currently not used anyway. A meeting with Tobias will be needed to do it properly.

There's no collection on schedules but only on charging profiles.

Reply:
> > It's just a starting point to have refresh from the DB on the charging profile schedule. It's currently not used anyway. A meeting with Tobias will be needed to do it properly.
> 
> There's no collection on schedules but only on charging profiles.

I know, some changes filtering will be needed on the collection chargingprofiles to refresh only when the current viewed profile record is updated. A bit like transaction details refreshing. I've talked about how to properly implement it during the last Monday team meeting. I'm just putting in place all the change listeners needed to properly implement it.

Reply:
> I know, some changes filtering will be needed on the collection chargingprofiles to refresh only when the current viewed profile record is updated. A bit like transaction details refreshing. I've talked about how to properly implement it during the last Monday team meeting. I'm just putting in place all the change listeners needed to properly implement it.

We have to discuss this together before you start anything.
I'll call you tomorrow.